### ISSUE TITLE
[FWI] Error in Code Generated

### ISSUE THREAD

**Issue by vmickus** [ufrn-lapps/devito#9]
When executing the FWI code (in sp4 project, file `03-fwi_man.py`), using the current solution in `dev` branch. The following error happens:

```
/tmp/devito-jitcache-uid1002/2cd113254badbcf5ba110d727022dd97dcbd82c3_ops.cpp: In function ‘int Forward(dataobj*, float, float, float, dataobj*, dataobj*, dataobj*, dataobj*, dataobj*, dataobj*, int, int, int, int, int, int, int, int, int, int, profiler*)’:
/tmp/devito-jitcache-uid1002/2cd113254badbcf5ba110d727022dd97dcbd82c3_ops.cpp:72:105: error: no matching function for call to ‘ops_decl_dat(ops_block_core*&, int, int [2], int [2], int [2], int [2], float (* __restrict__)[*(damp_vec->dataobj::size + 4)], const char [6], const char [5])’
   damp_dat = ops_decl_dat(block, 1, damp_dim, damp_base, damp_d_m, damp_d_p, &(damp[0]), "float", "damp");
```

This is just the first error of other similar.
Executed on sasha.

**Comment by vmickus** [comment]
From trello board, we have the following solution proposition:

Use a NULL variable instead of 'u' value:

`ops_decl_dat(block, 1, u_dim, u_base, u_d_m, u_d_p, (float ** ) NULL, "float", "utime0");`

**Comment by vmickus** [comment]
Fixed in PR #12
